AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1488,7 +1488,7 @@ bournemouthecho.co.uk,eveningtimes.co.uk,greenocktelegraph.co.uk,heraldscotland.
 oneplus.com,tickets.ua,woman.ru,elle.ru,marieclaire.ru,europaplus.ru,krossovki.com,belretail.by,thecollegefix.com,femmie.ru,prostofinovo.ru,mauijim.com,netpeak.net,wday.ru,buhgalteria.ru,ellegirl.ru,gameslog.ru,gizmag.com,healthhacks.ru,klerk.ru,menshealth.com,rbc.ru,snce.ru,turizm.ru,wegame.com.ua##.subscribe
 !
 mk.ru##.article__subscription
-bloggif.com###right > div.sky + h2
+bloggif.com#?##right > h2:contains(/จดหมายข่าว|бюлетина|ewsletter|δελτίο|Boletín|hírlevél|ニュースレター|naujienas|Nieuwsbrief|рассылку|nyhetsbrevet|Bültenimizi|接收简讯/)
```

Comment:
I mean something like that, without extended CSS
`#right > #rightTools + div.sky + h2`